.github/workflows/codeql-analysis.yml

```diff
@@ -14,6 +14,9 @@ name: "CodeQL"
 on:
   push:
     branches: [ master ]
+    # Workflows triggered by Dependabot on the "push" event run with read-only access
+    # Code Scanning requires write access, so ignore dependabot branches
+    branches-ignore: [dependabot/**]
 #  pull_request:
 #    # The branches below must be a subset of the branches above
 #    branches: [ master ]
```

Comment:
@dannyroberts is there a reason to leave this commented out as opposed to removing it entirely?

Reply:
I had vaguely thought we'd add this back when we wanted to incorporate this tool into pr review. Currently we're just using the run on master to see what comes up